### PR TITLE
Kiali tempo tutorial 

### PR DESCRIPTION
**Pull request by josunect**
New tutorial to guide step by step how to install Kiali with Grafana tempo in a local environment with Kubernetes using the tempo operator. 

Epic Ref. [#5850](https://github.com/kiali/kiali/issues/5850)

https://deploy-preview-686--kiali.netlify.app/docs/tutorials/tempo/

Changes in hack script (Run kiali): https://github.com/kiali/kiali/pull/6468

### CODE REVIEW

File: content/en/docs/Tutorials/tempo/01-Introduction.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+title: "Introduction"
+description: "Kiali and tempo integration introduction and prerequisites"
```
Comment (leandroberetta) [inline content/en/docs/Tutorials/tempo/01-Introduction.md]: Capitalize "tempo"

File: content/en/docs/Tutorials/tempo/01-Introduction.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+### Introduction
+
+Kiali uses [Jaeger](https://kiali.io/docs/configuration/p8s-jaeger-grafana/jaeger/) as a default distributed tracing backend. In this tutorial, we will replace it for [Grafana Tempo](https://grafana.com/docs/tempo/next/).
+
+We will setup a local environment in minikube, and install Kiali with Tempo as a distributed backend. This is a simplified architecture diagram:
+
+![Kiali Tempo Architecture](/images/tutorial/tempo/kiali-tempo.png "Kiali Tempo integration architecture")
+
+* We will install Tempo with the Tempo Operator and enable Jaeger query frontend to be compatible with Kiali in order to query traces.
+* We will setup Istio to send traces to the Tempo collector using the zipkin protocol. It is enabled by default from version > 3.0 of the Tempo Operator.
```
Comment (leandroberetta) [inline content/en/docs/Tutorials/tempo/01-Introduction.md]: I would say from version 3.0 or higher

File: content/en/docs/Tutorials/tempo/02-kiali-tempo-integration.md [hunk 3]
```diff
@@ -0,0 +1,244 @@
...
+minikube start
+```
+
+It is a requirement to have cert-manager installed: 
+
+```
+kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.12.0/cert-manager.yaml
+```
+
+Install the operator. It is important to download a version >=3.0. In previous versions, the zipkin collector was not exposed, there was no way to change it as it was not defined in the CRD.
```
Comment (leandroberetta) [inline content/en/docs/Tutorials/tempo/02-kiali-tempo-integration.md]: Similar here, 3.0 or higher.

File: content/en/docs/Tutorials/tempo/02-kiali-tempo-integration.md [hunk 4]
```diff
@@ -0,0 +1,244 @@
...
+      name: api
+    - port: 9001
+      targetPort: 9001
+      protocol: TCP
+      name: console
+  selector:
+    app: minio
+```
+
+And apply yaml:
```
Comment (leandroberetta) [inline content/en/docs/Tutorials/tempo/02-kiali-tempo-integration.md]: And apply the yaml

File: content/en/docs/Tutorials/tempo/02-kiali-tempo-integration.md [hunk 5]
```diff
@@ -0,0 +1,244 @@
...
+  template:
+    queryFrontend:
+      jaegerQuery:
+        enabled: true
+        ingress:
+          type: ingress
+EOF
+```
+
+(Optional) We can check if all the deployments have started correctly, and the services distributor has the port 9411 and the query frontend 16686:
```
Comment (leandroberetta) [inline content/en/docs/Tutorials/tempo/02-kiali-tempo-integration.md]: Here we can say "It's not required" or "as an optional step"

File: content/en/docs/Tutorials/tempo/02-kiali-tempo-integration.md [hunk 6]
```diff
@@ -0,0 +1,244 @@
...
+```
+kubectl get all -n tempo
+```
+![Tempo Services](/images/tutorial/tempo/tempo-services.png "Tempo Services")
+
+(Optional) We can test if minio is working with a batch job to send some traces, in this case, to th open telemetry collector:
```
Comment (leandroberetta) [inline content/en/docs/Tutorials/tempo/02-kiali-tempo-integration.md]: "to the open telemetry..."

**Review by leandroberetta — CHANGES_REQUESTED** [review]
Nice tutorial! Very interesting as I didn't know much about Tempo. I have just some ultra minor comments :)

File: content/en/docs/Tutorials/tempo/01-Introduction.md (hunk 1 above)
Comment (josunect) [reply content/en/docs/Tutorials/tempo/01-Introduction.md]: Updated

File: content/en/docs/Tutorials/tempo/01-Introduction.md (hunk 2 above)
Comment (josunect) [reply content/en/docs/Tutorials/tempo/01-Introduction.md]: Thanks! I was thinking how to put it :)

File: content/en/docs/Tutorials/tempo/02-kiali-tempo-integration.md (hunk 3 above)
Comment (josunect) [reply content/en/docs/Tutorials/tempo/02-kiali-tempo-integration.md]: Updated

File: content/en/docs/Tutorials/tempo/02-kiali-tempo-integration.md (hunk 4 above)
Comment (josunect) [reply content/en/docs/Tutorials/tempo/02-kiali-tempo-integration.md]: Updated

File: content/en/docs/Tutorials/tempo/02-kiali-tempo-integration.md (hunk 5 above)
Comment (josunect) [reply content/en/docs/Tutorials/tempo/02-kiali-tempo-integration.md]: Ok, updated.

File: content/en/docs/Tutorials/tempo/02-kiali-tempo-integration.md (hunk 6 above)
Comment (josunect) [reply content/en/docs/Tutorials/tempo/02-kiali-tempo-integration.md]: Changed.

**Review by leandroberetta — APPROVED** [review]
Looks good!

**Comment by josunect** [comment]
> Looks good!

Thanks for the review! 

I will merge, as the 404 errors from the navigation looks like they are just from the new pages.

**Comment by aljesusg** [comment]
Congrats @josunect !!! And thanks @leandroberetta  for review !

**Pull request merged**